src/sentry/static/sentry/app/views/stream/utils.js

```diff
@@ -1,15 +1,23 @@
+// remove leading and trailing whitespace and remove double spaces
+function formatQueryString(qs) {
+  return qs.replace(/^\s+|\s+$/g, '').replace(/\s+/g, ' ');
```

Comment:
wouldn't it be a bit more clear if we used `trim()`

Reply:
Possibly, just copied what's already there which seems not too crazy though :)

Reply:
Oh wait, you mean native trim(), yeah, i'll update. (somehow thought you were referring to lodash trim earlier)